src/app/app.component.ts

```diff
@@ -55,6 +61,7 @@ export class AppComponent implements OnInit {
 
 
   createWallet() {
+    this.dialog.open(ModalsComponent, {disableClose: true, width: '100%', height: '100%'});
```

Comment:
Please make sure we only make one service call, and not a dialogue and modal service call...

Reply:
We need to get rid of this

Reply:
Yes I am going to open that material model inside  of Modal service